README.md

```diff
@@ -76,4 +76,4 @@ Phalcon is open source software licensed under the BSD 3-Clause License.
 
 Copyright © 2011-present, Phalcon Team.
 
-See the [LICENSE.txt](https://github.com/phalcon/cphalcon/blob/master/LICENSE.txt) file for more. Additional licenses of packages that Phalcon uses, is inspired by or has adapted is located in the [3rdparty/licenses][https://github.com/phalcon/cphalcon/blob/master/3rdparty/licenses] directory.
+See the [LICENSE.txt](https://github.com/phalcon/cphalcon/blob/master/LICENSE.txt) file for more. Additional licenses of packages that Phalcon uses are inspired by or have adapted is located in the [3rdparty/licenses][https://github.com/phalcon/cphalcon/blob/master/3rdparty/licenses] directory.
```

Comment:
Please revert this. 

Explanation: Additional licenses that Phalcon uses, is inspired by (Phalcon that is), or has adapted (again Phalcon has adapted)

Reply:
Should read
> Additional licenses of packages that Phalcon uses inspired by or adapted **are** located in the [3rdparty/licenses][https://github.com/phalcon/cphalcon/blob/master/3rdparty/licenses] directory.

Also if we used LICENSE.md we'd get the pretty license tldr from GitHub as well.